sign/rpmgensig.cc

```diff
@@ -370,6 +372,7 @@ static void deleteSigs(Header sigh)
     headerDel(sigh, RPMSIGTAG_DSA);
     headerDel(sigh, RPMSIGTAG_RSA);
     headerDel(sigh, RPMSIGTAG_PGP5);
+    headerDel(sigh, RPMSIGTAG_OPENPGP);
```

Comment:
@pmatilai  Presumably this means that older versions of RPM will not be able to clear signatures stored in `RPMSIGTAG_OPENPGP`?

Reply:
Indeed. We should backport that much to 4.20 at least.